onyo/lib/commands.py

```diff
@@ -87,25 +125,44 @@ def onyo_cat(repo: OnyoRepo, paths: Iterable[Path]) -> None:
     assets_valid = validate_yaml(set(paths))
     # open file and print to stdout
     for path in paths:
-        # TODO: Probably better to simply print dict_to_yaml(repo.get_asset_content(path)) - no need to distinguish
-        #       asset and asset dir at this level. However, need to make sure to not print pointless empty lines.
+        # TODO: Probably better to simply print
+        #       `dict_to_yaml(repo.get_asset_content(path))` - no need to
+        #       distinguish asset and asset dir at this level. However, need to
+        #       make sure to not print pointless empty lines.
         f = path / OnyoRepo.ASSET_DIR_FILE if repo.is_asset_dir(path) else path
         ui.print(f.read_text(), end='')
     if not assets_valid:
         raise OnyoInvalidRepoError("Invalid assets")
 
 
-def onyo_config(repo: OnyoRepo, config_args: list[str]) -> None:
+def onyo_config(repo: OnyoRepo,
+                config_args: list[str]) -> None:
+    """Run the command `onyo config` to configure the onyo repository-wide
+    config options.
```

Comment:
Apart form the "run ...", this is not only about configuring as in setting a config. You can also get it.

```suggestion
    """Interface the configuration of an onyo repository.
```